modules/gltf/extensions/gltf_light.cpp

```diff
@@ -182,6 +182,7 @@ Light3D *GLTFLight::to_node() const {
 		return nullptr;
 	}
 	light->set_color(color.linear_to_srgb());
+	light->set_param(Light3D::PARAM_ATTENUATION, 2.0);
```

Comment:
Should we do anything with this value on export? We want to ensure that if a glTF file is exported from Godot, it will be imported again with at least roughly the same appearance.